extensions/dacpac/src/utils.ts

```diff
@@ -22,14 +23,21 @@ export function getPackageInfo(packageJson: any): IPackageInfo | undefined {
 }
 
 /**
- * Map an error message into a short name for the type of error.
- * @param msg The error message to map
+ * Get file size from the file stats using the file path uri
+ * @param uri The file path
  */
-export function getTelemetryErrorType(msg: string): string {
-	if (msg && msg.indexOf('Object reference not set to an instance of an object') !== -1) {
-		return 'ObjectReferenceNotSet';
+export async function getFileSize(uri: string): Promise<number | undefined> {
+	const stats = await getFileStatus(uri);
+	return stats ? stats.size : undefined;
+}
+
+async function getFileStatus(path: string): Promise<fs.Stats | undefined> {
+	try {
+		const stats = await fs.promises.stat(path);
+		return stats;
 	}
-	else {
-		return 'Other';
+	catch (e) {
+		// 'ENOENT' - Error NO ENTity
```

Comment:
There's a bunch of different types of errors that can happen here so I'd just have the comment say something about this purposely ignoring errors